package.json

```diff
@@ -22,7 +22,6 @@
     "prism": "^1.27.0"
   },
   "dependencies": {
-    "@department-of-veterans-affairs/css-library": "^0.11.1",
```

Comment:
I'm not sure why this is needed here. We don't run `yarn build` in the repo's root folder as part of our [development build steps](https://github.com/department-of-veterans-affairs/component-library?tab=readme-ov-file#running-build-via-storybook). We run it in `web-components`, `core`, `react-components`, and `storybook`. 

As far as I can tell, css-library is just needed in the [web-components package.json](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/package.json#L30) so I'm going to clean this up to avoid confusion (it's me, I was the confused one).